Lesson_15/geometric_homework.py

```diff
@@ -0,0 +1,24 @@
+import math
+
+class Rhombus:
+    def __init__(self, side_a, angle_a):
+        self.setattr('side_a', side_a)
+        self.setattr('angle_a', angle_a)
+
+    def setattr(self, name, value):
```

Comment:
It is better to use __setattr__ dunder method for setting attributes. In case you get __dict__ list of object attributes - they could not been initialized yet before setting.